### DISCUSSION TITLE
Nice new sound features

### DISCUSSION THREAD

**Discussion by CptBayer** [flybywiresim/aircraft#6793] [General]
Hi dev team,

by the way, one more time many thanx for your fantastic work. I like the new sound features :-) .... unlocked door - galley noise :-) .. and all others, too.

greets Klaus

**Comment by hotshotp** [comment]
Thanks! Much appreciated

**Reply by DedaL54** [reply]
When will the stable version be upgraded with new details that development versions have?

**Reply by hotshotp** [reply]
When it's ready. That's really it, no concrete dates

**Reply by tracernz** [reply]
> When will the stable version be upgraded with new details that development versions have?

You can simply install the dev version. What's in it will be the next "stable".

**Reply by DedaL54** [reply]
Okay, thanks for the reply. I think I will patiently stay on the stable version!  :)
